kodiak/main.py

```diff
@@ -37,7 +48,7 @@
 )
 
 app = FastAPI()
-app.add_middleware(SentryMiddleware)
+app.add_middleware(SentryAsgiMiddleware)
```

Comment:
What's the difference between these middlewares?

Reply:
The old one was a third party package, the new one is integrated into the sentry sdk